web/client/components/widgets/widget/TableWidget.jsx

```diff
@@ -67,6 +68,8 @@ module.exports = ({
                 columnSettings={columnSettings}
                 pageEvents={pageEvents}
                 virtualScroll={virtualScroll}
+                enableColumnFilters
+                filterRenderers={getWidgetFilterRenderers(describeFeatureType, id)}
```

Comment:
create an enhancer that adds filterRenderers prop, you have all the information you need to create filter renderers from prop, no need to do strange things (also attributeFilter should be available at this scope)